### PR TITLE
Switch to new crypto APIs for OTP 23

### PR DESCRIPTION
**Pull request by Vagabond**

### CODE REVIEW

**Comment by Vagabond** [comment]
There's something wrong with the disterl tests on OTP 23

File: rebar.config [hunk 1]
```diff
@@ -29,7 +29,7 @@
 {profiles, [
             {test, [
                {deps, [
-                       {relcast, {git, "https://github.com/helium/relcast.git", {branch, "master"}}}
+                       {relcast, {git, "https://github.com/helium/relcast.git", {branch, "adt/otp-23"}}}
```
Comment (JayKickliter) [inline rebar.config:32]: Should we block this PR on https://github.com/helium/relcast/pull/41 and update this to a hash when that merges?

Comment (Vagabond) [reply rebar.config:32]: it's a test only dep, but yes

Comment (evanmcc) [reply rebar.config]: fixed this post relcast upgrade.

**Comment by evanmcc** [comment]
looks like relcast picked up some lager debugging, so it needs the parse transform not to crash

**Review by evanmcc — APPROVED** [review]

**Comment by evanmcc** [comment]
Crap.  I thought that the CI here ran the tests and assumed that they passed on 23, but if they pass, it's on 22.  I'll look into it more, but we shouldn't upgrade to this until we figure out what the deal is.

**Pull request merged**